<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>System</class>
 <widget class="QWidget" name="System">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="currentText">
    <string>dog</string>
   </property>
   <item>
    <property name="text">
     <string>dog</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cat</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="stu_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>391</width>
     <height>131</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <column>
    <property name="text">
     <string>id</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>amount</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
